clarifai/models/model_serving/pb_model_repository.py

```diff
@@ -80,22 +82,30 @@ def build_repository(self, repository_dir: Path = os.curdir):
           continue
       # gen requirements
       with open(os.path.join(repository_path, "requirements.txt"), "w") as f:
-        f.write("clarifai>9.5.3\ntritonclient[all]")  # for model upload utils
+        f.write("clarifai>9.10.4\ntritonclient[all]")  # for model upload utils
 
     if not os.path.isdir(model_version_path):
       os.mkdir(model_version_path)
     if not os.path.exists(os.path.join(model_version_path, "__init__.py")):
       with open(os.path.join(model_version_path, "__init__.py"), "w"):
         pass
-    # generate model.py & inference.py modules
-    self._module_to_file(pb_model, filename="model.py", destination_dir=model_version_path)
-    self._module_to_file(inference, filename="inference.py", destination_dir=model_version_path)
+    # generate model.py
+    model_py_path = os.path.join(model_version_path, "model.py")
+    self._module_to_file(pb_model, model_py_path, func=None)
+
+    # generate inference.py
+    def inference_insert_model_type_func(x):
+      x = x.replace("""#config = get_model_config("clarifai-model-type")""",
+                    f"""config = get_model_config("{self.model_config.model_type}")""")
+      x = x.replace("#@config.inference.wrap_func", "@config.inference.wrap_func")
+      return x
+
+    inference_py_path = os.path.join(model_version_path, "inference.py")
+    self._module_to_file(inference, inference_py_path, inference_insert_model_type_func)
+
     # generate test.py
+    def insert_model_type_func(x):
+      return x.replace("clarifai-model-type", self.model_config.model_type)
```

Comment:
It would be good to also update test.py to use similarly named `MODEL_TYPE_PLACEHOLDER` and then you only need one helper function for inference.py and test.py